kettle-columnstore-bulk-exporter-plugin/src/main/java/com/mariadb/columnstore/api/kettle/KettleColumnStoreBulkExporterStepMeta.java

```diff
@@ -87,9 +88,20 @@
   static {
       String nativeLib = "";
   try {
-      String jar = KettleColumnStoreBulkExporterStepMeta.class.getProtectionDomain().getCodeSource().getLocation().toURI().getPath();
-      String jarPath = jar.substring(0, jar.lastIndexOf(File.separator));
-      nativeLib = jarPath + File.separator + "lib" + File.separator + "libjavamcsapi.so";
+      String jar = new URI(KettleColumnStoreBulkExporterStepMeta.class.getProtectionDomain().getCodeSource().getLocation().toString().replace(" ", "%20")).getPath();
+      System.err.println(jar);
+      String jarPath = jar.substring(0, jar.lastIndexOf("/"));
+      if(System.getProperty("os.name").indexOf("nux") >= 0){
```

Comment:
I think it would be safer to invert the logic here and make it if <windows> then .. else .. (implied unix default).